VERSION

```diff
@@ -1 +1 @@
-0.16.0-dev
\ No newline at end of file
+0.17.0-dev
```

Comment:
I think you might be right @metalmatze 

Maybe 0.18.0-dev makes more sense? (:

Reply:
Updating!